src/commands/delete-excsessive-pvc

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+#set -x
+
+readonly PVC_SELECTOR=${1:-app=collection-sumologic-fluentd-logs}
+readonly HPA_NAME=${2:-collection-sumologic-fluentd-logs}
+readonly NAMESPACE=${3:-sumologic}
+
+function get_sorted_pvcs() {
+  kubectl -n "$NAMESPACE"  get pvc --selector="$PVC_SELECTOR" --no-headers \
+    | awk '{print $1}' \
+    | sort
+}
+
+function max_number() {
+  local num1="$1"
+  local num2="$2"
+
+  [[ $num1 -gt $num2 ]] \
+    && echo "$1" \
+    || echo "$2"
```

Comment:
could we use `if` `else` for readability?

Reply:
Done.